tests/integration/helpers.bash

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
-# bats-core v1.2.1 defines BATS_RUN_TMPDIR
-if [ -z "$BATS_RUN_TMPDIR" ]; then
+set -u
```

Comment:
One problem at a time and all that, but this probably ought to consider getting at least `-e` as well (if not `-o pipefail` too). :see_no_evil:

(Also a much less trivial change than it appears on the face, just like enabling `set -u` :sweat_smile:)

Reply:
In fact, `bats` sets `test -e` for test cases (i.e. anything that is called from `@test xxx {...}`), so most of the code is `set -e`-friendly already, it's only the code that is called from `setup/cleanup` that is not.

I do have that in mind, but thanks for reminding. Heck, let's open an issue about it. Here we go: https://github.com/opencontainers/runc/issues/3399 (oh, and feel free to tackle it, we need more eyes and hands in this repo)